app/controllers/phases_controller.rb

```diff
@@ -0,0 +1,64 @@
+class PhasesController < ApplicationController
```

Comment:
I think phase controller route should be nested with the project lead. The route should be like 
/leads/1/phases
/leads/2/phases/new
/leads/3/phases/4
...

Please fix this controller in the above mentioned way.